seqr/views/apis/analysis_group_api.py

```diff
@@ -55,6 +55,7 @@ def update_analysis_group_handler(request, project_guid, analysis_group_guid=Non
     valid_families = set()
 
     def _validate_families(request_json):
+        request_json.pop('uploadedFamilyIds', None)
```

Comment:
Is it possible to remove the field from the request body when it's created instead?

Reply:
not easily, its would involve a pretty messy update to a shared utility helper